<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>773</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(212, 213, 229);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pressButton10">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>240</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>10</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton9">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton8">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton7">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>280</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton6">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>320</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton5">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>320</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton4">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>360</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton3">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>360</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>400</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressButton1">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>400</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="holdOpen">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>460</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;
}

QPushButton:hover {
background-color: orange
}</string>
    </property>
    <property name="text">
     <string>&lt; &gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="holdClose">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>460</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;
}

QPushButton:hover {
background-color: orange
}</string>
    </property>
    <property name="text">
     <string>&gt; &lt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressCallButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>500</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Call</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pressHelpButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>41</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(180, 182, 203);
border-style: outset;
border-width: 2px;
border-radius: 16px;
border-color: black;</string>
    </property>
    <property name="text">
     <string>Help</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_4" native="true">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>170</y>
      <width>141</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style: outset;
border-width: 2px;
border-color: rgb(180, 182, 203);
background-color: rgb(212, 213, 229);</string>
    </property>
    <widget class="QLCDNumber" name="displayFloor">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 0, 0);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="intUpArrow">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
font: 32pt;</string>
    </property>
    <property name="text">
     <string> ↑</string>
    </property>
   </widget>
   <widget class="QLabel" name="intDownArrow">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 32pt;
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string> ↓</string>
    </property>
   </widget>
   <widget class="QPushButton" name="intLeft">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>41</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(121, 129, 161);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="intRight">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>41</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(121, 129, 161);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushBackButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(188, 0, 30);</string>
    </property>
    <property name="text">
     <string>BACK</string>
    </property>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>10</y>
      <width>411</width>
      <height>691</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);</string>
    </property>
    <widget class="QPushButton" name="pushExit">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>320</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(68, 68, 68);</string>
     </property>
     <property name="text">
      <string>Exit Elevator</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>831</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="carpet.qrc">:/carpet/Carpet.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="changeView">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>570</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(121, 129, 161);</string>
    </property>
    <property name="text">
     <string>Change Floor View</string>
    </property>
   </widget>
   <zorder>widget_2</zorder>
   <zorder>widget_4</zorder>
   <zorder>pressButton10</zorder>
   <zorder>pressButton9</zorder>
   <zorder>pressButton8</zorder>
   <zorder>pressButton7</zorder>
   <zorder>pressButton6</zorder>
   <zorder>pressButton5</zorder>
   <zorder>pressButton4</zorder>
   <zorder>pressButton3</zorder>
   <zorder>pressButton2</zorder>
   <zorder>pressButton1</zorder>
   <zorder>holdOpen</zorder>
   <zorder>holdClose</zorder>
   <zorder>pressCallButton</zorder>
   <zorder>pressHelpButton</zorder>
   <zorder>intUpArrow</zorder>
   <zorder>intDownArrow</zorder>
   <zorder>intLeft</zorder>
   <zorder>intRight</zorder>
   <zorder>pushBackButton</zorder>
   <zorder>label</zorder>
   <zorder>changeView</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="carpet.qrc"/>
 </resources>
 <connections/>
</ui>
